RecoHGCal/TICL/python/HADStep_cff.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-from RecoHGCal.TICL.TICLSeedingRegions_cff import ticlSeedingGlobal
+from RecoHGCal.TICL.TICLSeedingRegions_cff import ticlSeedingGlobal, ticlSeedingGlobalHFNose, ticlSeedingForHFHFNose
```

Comment:
I cannot find where `ticlSeedingForHFHFNose` is defined.
Possibly you meant `ticlSeedingTrkHFNose`?

Reply:
the test failed because the ticlSeedingForHFHFNose wasn't committed; I did now.
ticlSeedingByHFHFNose allows to seed from a HFcell; but the iteration is not build as would be better to build the trackster outside-in.

Reply:
for the hadrons I foresee 3 set of iterations:

1. early shower for charger particle: track seeded + short trackster
2. late shower: HFseeded  with outside-in trackster
3. all the rest will be taken from the self seeded